src/main/kotlin/com/neva/gradle/fork/config/properties/PropertyDefinition.kt

```diff
@@ -0,0 +1,72 @@
+package com.neva.gradle.fork.config.properties
+
+class PropertyDefinitions {
+  private val definitions = mutableMapOf<String, PropertyDefinition>()
```

Comment:
empty line around fields

Reply:
mine and Cognifide's is matching each other, I have no problem with that :>

Intellij default + blank line around fields :D